MAUI/Toolbar/toolbar-items.md

```diff
@@ -0,0 +1,1903 @@
+            toolbar.HeightRequest = 56;
+
+            ObservableCollection<BaseToolbarItem> toolbarItems = new ObservableCollection<BaseToolbarItem>
+            {
+                new SfToolbarItem
+                {
+                    Name = "Bold",
+                    Size = new Size(60, 40),
+                    Icon = new FontImageSource { Glyph = "\uE770", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "Underline",
+                    Size = new Size(90, 40),
+                    Icon = new FontImageSource { Glyph = "\uE762", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "Italic",
+                    Size = new Size(60, 40),
+                    Icon = new FontImageSource { Glyph = "\uE771", FontFamily = "MauiMaterialAssets" }
+                },
+                new SeparatorToolbarItem {},
+                new SfToolbarItem
+                {
+                    Name = "AlignLeft",
+                    Size = new Size(90, 40),
+                    Icon = new FontImageSource { Glyph = "\uE751", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "AlignRight",
+                    Size = new Size(100, 40),
+                    Icon = new FontImageSource { Glyph = "\uE753", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "AlignCenter",
+                    Size = new Size(100, 40),
+                    Icon = new FontImageSource { Glyph = "\uE752", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "AlignJustify",
+                    Size = new Size(100, 40),
+                    Icon = new FontImageSource { Glyph = "\uE74F", FontFamily = "MauiMaterialAssets" }
+                },
+                new SeparatorToolbarItem {},
+                new SfToolbarItem
+                {
+                    Name = "Undo",
+                    Size = new Size(100, 40),
+                    Icon = new FontImageSource { Glyph = "\uE744", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "Redo",
+                    Size = new Size(100, 40),
+                    Icon = new FontImageSource { Glyph = "\uE745", FontFamily = "MauiMaterialAssets" }
+                }
+            };
+
+            toolbar.Items = toolbarItems;
+            this.Content = toolbar;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Custom View
+
+The Toolbar control provides support to customize the items of the toolbar with any type of views. You can add the views to the toolbar using the `View` property.
```

Comment:
You can add custom views, such as buttons, checkboxes, entries, etc., to the toolbar using the View property.